addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -362,6 +362,11 @@ static func get_built_in_categories(_class_name: String) -> Array[BlockCategory]
 				b.signal_name = "body_%s" % [verb]
 				block_list.append(b)
 
+			var b = BLOCKS["statement_block"].instantiate()
+			b.block_format = "Set Physics Position {position: VECTOR2}"
+			b.statement = "PhysicsServer2D.body_set_state(get_rid(),PhysicsServer2D.BODY_STATE_TRANSFORM,Transform2D.IDENTITY.translated({position}))"
```

Comment:
I wonder why this is not needed in Moddable Pong.

I think because what we do for respawning the ball is (from memory):
- Duplicate the node
- Queue free the old one
- Set a random position to the new one, and finally call add_child

Reply:
Yes, I think setting the position this way is fine too, since we don't have node instantiation yet. This seemed like the simplest way.

Also, we should add random blocks in the future! I remember using them very often in Scratch, and they introduced me to the world of procedural generation :)